source/openapi-admin-v3.yaml

```diff
@@ -5600,7 +5700,7 @@ components:
         - type
         - config
       oneOf:
-        - $ref: "#/components/schemas/AtlasCluster"
+        - $ref: "#/components/schemas/AppConstructor"
```

Comment:
This one should still be `AtlasCluster`.

`AppConstructor` reference `DataSource` so this is circular as-is.